src/test/java/com/flightstats/hub/dao/aws/ClusterContentServiceTest.java

```diff
@@ -17,6 +17,7 @@
 import com.flightstats.hub.util.HubUtils;
 import com.flightstats.hub.util.TimeUtil;
 import lombok.SneakyThrows;
+import org.joda.time.DateTime;
```

Comment:
Java 8 date library should be favored here. Performance is better with Java 8 dates compared to joda-time.

Reply:
I believe we have a task to migrate the hub off joda and onto Java 8 native stuff, but it'll be a pretty big and sensitive change. I'll check to make sure we're tracking that debt.

Reply:
Yeah, joda's heavily used throughout hub, so leaving this for now for consistency.  Agree it'd be nice to switch over though.